react-hot-button/my-app/src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import './App.css';
+
+const label = 'Hot Button';
+export default function HotButton() {
```

Comment:
In the future, as things get more complex, you'll want to keep App nice and simple and instead put the component in a separate file.